airmozilla/starred/tests/test_views.py

```diff
@@ -227,3 +227,53 @@ def test_anonymous_starred_events(self):
         ok_(event1.title in response.content)
         ok_(event2.title in response.content)
         ok_(event3.title not in response.content)
+
+    def test_event_pagination(self):
+        url = self.home_url
+        user = self._login(username='lisa')
+
+        events = []
+
+        for i in range(25):
+            event = self.create_event("Event Title %d" % (i + 1))
+            StarredEvent.objects.create(user=user, event=event)
+            events.append(event.id)
+
+        response = self.client.get(url)
+        eq_(response.status_code, 200)
+
+        for i in range(len(events)):
+            match = 'data-id="%d"' % events[i]
+            if i < 10:
+                self.assertTrue(match in response.content, i)
```

Comment:
Instead of `self.assertTrue` we use the "shorthand" tool which is `ok_()`.
And the anti-equivalent of `ok_(...)` is `ok_(not ...)`.